components/hyper_serde/lib.rs

```diff
@@ -138,7 +138,7 @@ impl<T> De<T> {
     /// Returns a new `De` wrapper
     #[inline(always)]
     pub fn new(v: T) -> Self {
-        De { v: v }
+        De { v }
```

Comment:
It looks like you have also included your changes from your other PR. Please remove these changes from the PR.